src/components/forms/Button/index.js

```diff
@@ -23,7 +23,9 @@ class Button extends React.Component {
       fullWidth,
       palette,
       outline,
+      showArrow,
```

Comment:
I was working in a component similar here: https://github.com/realworldreact/realworldreact/blob/develop/src/pages/home/_button.js
For the home buttons.

But I like your idea better.

For now, let's support you can show the arrow with a prop as you are doing and with the background primary.base and text color primary.contrastText. On hover the arrow should also highlight.